frontend/public/components/factory/list.tsx

```diff
@@ -385,7 +385,7 @@ export const List = connect(
     static propTypes = {
       data: PropTypes.array,
       unfilteredData: PropTypes.array,
-      AllItemsFilteredMsg: PropTypes.func,
```

Comment:
This prop was misnamed... It's used when there's no data at all, not when data is filtered.